client/my-sites/plugins/plugin-details.jsx

```diff
@@ -250,6 +252,20 @@ function PluginDetails( props ) {
 
 	const showPlaceholder = existingPlugin === 'unknown';
 
+	if ( props.isJetpackCloud ) {
+		return (
+			<PluginDetailsV2
+				showPlaceholder={ showPlaceholder }
+				selectedSite={ selectedSite }
+				fullPlugin={ fullPlugin }
+				sitesWithPlugins={ sitesWithPlugins }
+				isMarketplaceProduct={ isMarketplaceProduct }
+				isWpcom={ isWpcom }
+				{ ...props }
+			/>
+		);
+	}
```

Comment:
If we define the `plugins/plugin-details-layout` in the `jetpack-cloud-development.json` file, I think we can remove this check and select the right component according to `legacyVersion` value.

Reply:
I think we will still need this check since this component handles both the new and legacy version.